src/test/scala/scalismo/faces/image/CorrelationFilterTest.scala

```diff
@@ -42,7 +42,7 @@ class CorrelationFilterTest extends FacesTestSuite with PrivateMethodTester {
 
     it("yields a symmetric image (I==I.t) for a separable kernel (regression test for access modes in separable filters)") {
       val checkers = PixelImage(15, 15, (x,y) => if((x+y)%2==0) 1.0 else 0.0 )
-      val blurKernel = PixelImage(PixelImageDomain(3, 1), IndexedSeq(0.25, 0.5, 0.25))
+      val blurKernel = PixelImage(PixelImageDomain(3, 1), IndexedSeq(0.25, 0.5, 0.25).toIndexedSeq)
```

Comment:
Is this really needed in scala 2.13?

Reply:
I have not fully understood what is happening here. I suppose that it is an incompatibility between a mutable and immutable IndexedSeq. There should be a better way to solve this problem, but it might involve changing a type in PixelImage (which I did not dare to do).